### DISCUSSION TITLE
Is there a debug mode configuration for Intellij script?

### DISCUSSION THREAD

**Discussion by Fle9ma** [maciejwalkowiak/just#44] [Q&A] [answered]
I followed [this](https://just.maciejwalkowiak.com/docs/intellij-idea/) Intellij configuration tutorial and its works amazingly. But I was wondering how is possible to configure and run the service **just** in debug mode. I was searching in the documentation and couldn't find how to set up debug mode.

Thanks and have a nice day 😊

**Comment by maciejwalkowiak** [comment, marked as answer]
Hi @Fle9ma! Unfortunately not at this stage. I've done quite a few of POCs and having debug mode that survives application restart is not trivial. So I am still researching options.
